.github/ISSUE_TEMPLATE/doc-issue.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Documentation/tutorial notebooks
+about: Let us know what would make the GPflow documentation better
+---
+
+# Documentation/tutorial notebooks
+
+*Is there anything missing in the docs?*
+
+*Are there any mistakes in the docs?*
```

Comment:
if we're treating mistakes in the docs as a documentation issue not a bug report, we should explain that in the bug report template

Reply:
I don't think it matters too much - this is just a less-heavy version of the bug report - if someone is keen to fill in the more in-depth bug report, that works too, no?